app/views/point_of_interests/form_partials/_lunch_offers_form.html.erb

```diff
@@ -8,11 +8,9 @@
         <div class="card mb-4">
           <div class="d-flex flex-wrap justify-content-between card-header py-3 bg-dark text-white">
             <h4>Gericht</h4>
-            <input
-              type="button"
-              value="dieses Gericht entfernen"
-              class="remove btn btn-sm btn-danger"
-            />
+            <%= link_to "/",  class: "remove btn btn-sm btn-danger" do %>
```

Comment:
the link target shouldn't be `/` because this action should never link to the startpage, no matter what. a `#` would make more sense here

Reply:
done in cd774c50e816aefadac01b92f41858c62396ac57